Makefile

```diff
@@ -16,6 +14,13 @@ endif
 # tools. (i.e. podman)
 CONTAINER_TOOL ?= docker
 
+GIT_TAG ?= $(shell git describe --tags --dirty --always)
+# Image URL to use all building/pushing image targets
+IMAGE_REGISTRY ?= gcr.io/k8s-staging-lws
```

Comment:
We may need an official image address.

Reply:
That sounds good, I guess we need a PR similar to https://github.com/kubernetes/k8s.io/pull/5144